pdf/core/security/standard_r4.go

```diff
@@ -0,0 +1,349 @@
+/*
+ * This file is subject to the terms and conditions defined in
+ * file 'LICENSE.md', which is part of this source code package.
+ */
+
+package security
+
+import (
+	"bytes"
+	"crypto/md5"
+	"crypto/rand"
+	"crypto/rc4"
+	"errors"
+
+	"github.com/unidoc/unidoc/common"
+)
+
+var _ StdHandler = stdHandlerR4{}
```

Comment:
Why is this needed?

Reply:
This is a compile-time assertion for "stdHandlerR4 implements StdHandler". It is also useful to see what interface this type implements. I can remove them, but I personally find them rather useful.